website/content/api-docs/agent/index.mdx

```diff
@@ -720,17 +721,19 @@ The table below shows this endpoint's support for
 
 The corresponding CLI command is [`consul force-leave`](/commands/force-leave).
 
-### Parameters
+### Path Parameters
```

Comment:
@trujillo-adam and I agreed to split the HTTP API parameters into path parameters and query parameters sections, but I'll move that to a separate PR